colortv/docs/androidTVSDK.md

```diff
@@ -591,83 +591,214 @@ public class MainActivity extends Activity {
+<br /><br />
+
+
+#### **ID:** ctv_show
+* **View type:** Any view extending the View class (eg. `LinearLayout`, `ImageView`)
+* **Description:** Used to show views on focus and hide them on unfocus. Use any Layout if you would like to show more than one view.
+* **Animation:** Shows (`Visibility.VISIBLE`) the view and its child views on focus and hides them on unfocus.
+* **Device:** TV
+<br /><br />
+
+
+#### **ID:** ctv_autoPlayTimerContainer
+* **View type:** `FrameLayout`
+* **Description:** Used to inject countdown timer to automatically start playing the first recommended video
+* **Animation:** Hides on any interaction on items list or when timer reaches 0.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_videoPreviewContainer
+* **View type:** Any view extending the ViewGroup class (eg. `LinearLayout`, `RelativeLayout`)
+* **Description:** Used to inject our `VideoPlayer` for previews playback. Check our [custom layout](https://github.com/color-tv/android-SampleApp/blob/master/SampleApp/app/src/main/res/layout/custom_item_layout_2.xml) if you would like to oversize the preview video.
+* **Animation:** Plays the preview and hides the thumbnail on focus (only if the preview is available) and stops playing the preview and shows the thumbnail on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivVideoThumbnail
+* **View type:** `ImageView`
+* **Description:** Used to display the video thumbnail.
+* **Animation:** Hides on focus if there is preview available and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivAppLogo
+* **View type:** `ImageView`
+* **Description:** Used to display logo.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivPlayButton
+* **View type:** `ImageView`
+* **Animation:** Hides on focus and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivBlackMask
+* **View type:** `ImageView`
+* **Animation:** Hides on focus and shows on unfocus.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvTitle
+* **View type:** `TextView`
+* **Description:** Displays the video title.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDescription
+* **View type:** `TextView`
+* **Description:** Displays the video description.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvDuration
+* **View type:** `TextView`
+* **Description:** Displays the video duration.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_tvGenre
+* **View type:** `TextView`
+* **Description:** Defines how each genre should look like. Visibility must be set to `Visibility.GONE`.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_llGenres
+* **View type:** `LinearLayout`
+* **Description:** Used to contain genres, which will look like the `TextView` with id `ctv_tvGenre`.
+* **Device:** ALL
+<br /><br />
+
+
+#### **ID:** ctv_ivFavoriteIcon
+* **View type:** `ImageView`
+* **Description:** Shows `ctv_ivFavoriteContainer` on click.
+* **Device:** ALL
+
+</details>
```

Comment:
It would be useful if there would be an image with all elements marked and signed on a default layout.